cdap-ui/app/cdap/components/DataPrepConnections/index.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { Component } from 'react';
+import IconSVG from 'components/IconSVG';
+import classnames from 'classnames';
+import Match from 'react-router/Match';
+import Redirect from 'react-router/Redirect';
+import Link from 'react-router/Link';
+import FileBrowser from 'components/FileBrowser';
+import NamespaceStore from 'services/NamespaceStore';
+import T from 'i18n-react';
+
+require('./DataPrepConnections.scss');
+const PREFIX = 'features.DataPrepConnections';
+
+const RouteToHDFS = () => {
+  let namespace = NamespaceStore.getState().selectedNamespace;
+
+  return (
+    <Redirect to={`/ns/${namespace}/connections/browser`} />
+  );
+};
+
+const UploadPlaceholder = () => {
+  return (
+    <div>
+      <h3 className="text-xs-center">UPLOAD</h3>
```

Comment:
That's just a placeholder, that component will be completely removed when we have an upload page